pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -302,6 +295,9 @@ func (e *eksOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 			return e.setUnknown(cluster, apimgmtv3.ClusterConditionUpdated, "")
 		}
 		logrus.Infof("waiting for cluster EKS [%s] update failure to be resolved", cluster.Name)
+		if strings.Contains(failureMessage, "403") {
+			failureMessage = "cannot access EKS, check cloud credential"
```

Comment:
Why does this drop the original message? a 403 is much more explicit to being an auth issue, saying `cannot access EKS` seems ambiguous and leans more towards a networking issue.